tests/test_checkout.py

```diff
@@ -143,6 +160,23 @@ def test(self):
         self.assertFalse(os.path.exists(fname))
 
 
+class TestCheckoutSelectiveRemove(CheckoutBase):
+    def test(self):
+        # Use copy to test for changes in the inodes
+        main(['config', 'cache.type', 'copy'])
```

Comment:
```suggestion
        ret = main(['config', 'cache.type', 'copy'])
        self.assertEqual(ret, 0)
```

Reply:
@efiop, we can skip this assertion, as we are already testing the `config` command in `tests/test_config.py`.

Reply:
We do, but it doesn't save from making a mistake here. Better safe than sorry :slightly_smiling_face: